src/pages/dashboard/listings/[slug]/edit.tsx

```diff
@@ -26,6 +27,24 @@ function EditBounty({ slug }: Props) {
       if (bountyDetails.data.sponsorId !== userInfo?.currentSponsorId) {
         router.push('/dashboard/listings');
       } else {
+        const bounty = bountyDetails.data as Bounty;
+        if (
```

Comment:
the first if to make sure sponsors fill the all required fields of basics

without this, when using template, if sponsor saves at basics and comes back then he'll be placed at step 3 - description